encoder/vaapiencoder_hevc.cpp

```diff
@@ -1022,16 +1023,7 @@ Encode_Status VaapiEncoderHEVC::setParameters(VideoParamConfigType type, Yami_PT
     case VideoParamsTypeAVC: {
```

Comment:
How about providing a VideoParamsHEVC struct, like this:
typedef struct VideoParamsHEVC {
VideoParamsAVC avcParams;
...other hevc parameters.
}VideoParamsHEVC;